shared/login/register/passphrase/index.render.js.flow

```diff
@@ -7,7 +7,8 @@ export type Props = {
   onBack: () => void,
   onForgotPassphrase: () => void,
   waitingForResponse: boolean,
-  error?: ?string
+  error?: ?string,
+  username: string
```

Comment:
This should be a `?string` at least from above since you do pass it something that can be null.